contracts/voting-verifier/src/msg.rs

```diff
@@ -25,6 +26,12 @@ pub enum ExecuteMsg {
     #[permission(Any)]
     VerifyMessages(Vec<Message>),
 
+    #[permission(Any)]
+    VerifyMessageWithPayload {
+        message: Message,
+        payload: HexBinary,
+    },
```

Comment:
we should still retain the ability to do batching

Reply:
Done.